Ntuplizer/plugins/METsNtuplizer.cc

```diff
@@ -231,11 +237,60 @@ void METsNtuplizer::fillBranches( edm::Event const & event, const edm::EventSetu
   event.getByToken(metpuppiInputToken_, METspuppi_ );
 
   for (const pat::MET &met : *METspuppi_) {
-    nBranches_->METpuppi_et.push_back(met.et());
+    nBranches_->METpuppi_et.push_back(met.pt());
```

Comment:
For puppi met & PF met, pt() and et() return the same value. 
But for the MVA MET, et() is buggy and pt() is only correct. So I just wanted to make it uniform for all the MET values. 

PS.
missing ET is usually, missing ET "vector", i.e. not the - (scalar sum of ET). So I also prefer pt().